src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/config/LocalStackConfig.kt

```diff
@@ -20,7 +25,18 @@ class LocalStackConfig {
   }
 
   @Bean
-  fun localStackContainer(): LocalStackContainer {
+  fun elasticSearchContainer() : ElasticsearchContainer {
+    return ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch-oss:6.7.2")
```

Comment:
So why do we need localstack test containers and this special elastic search container?

Reply:
test container for localstack doesn't have ES